src/store/index.js

```diff
@@ -174,6 +180,12 @@ export default new Vuex.Store({
 			let errors = [];
 			let capabilities = cx.state.connection.capabilities();
 
+			// Note down federation things from capabilities
+			let federation = capabilities.listFederation();
+			if (Object.keys(federation).length > 0) {   // empty object = no federation -> don't commit (leaving default value)
```

Comment:
```suggestion
			if (federation.length > 0) {   // empty array = no federation -> don't commit (leaving default value)
```
listFederation always returns an array.